smoke/LeafDevice/details/Details.cs

```diff
@@ -156,6 +159,34 @@ protected async Task VerifyDataOnIoTHub()
             await eventHubClient.CloseAsync();
         }
 
+        static Task<MethodResponse> DirectMethod(MethodRequest methodRequest, object userContext)
+        {
+            Console.WriteLine("Leaf device received direct method call...");
+            return Task.FromResult(new MethodResponse((int)HttpStatusCode.OK));
+        }
+
+        protected async Task VerifyDirectMethod()
+        {
+            //User Service SDK to invoke Direct Method on the device.
+            ServiceClient serviceClient =
+                ServiceClient.CreateFromConnectionString(this.context.IotHubConnectionString);
+
+
+            //Call a direct method
+            using (var cts = new CancellationTokenSource(TimeSpan.FromSeconds(300)))
```

Comment:
I wonder if using the ConnectionTimeout and ResponseTimeout is a better check instead of the Cancellation token here..